### ISSUE TITLE
Why do Bootstrap plugins get built as a separate file?

### ISSUE THREAD

**Issue by silvenon** [yeoman/generator-webapp#590]
Seems like we could save an HTTP request there, if I'm not missing something.

**Comment by neilhem** [comment]
You mean bootstrap's built-in js plugins? The way how they are inserted into html file?

**Comment by silvenon** [comment]
@neilhem yeah, Bootstrap's JS plugins are inserted into a separate useref comment block, but it could just as well be a part of the `vendor.js` comment block, which would save an HTTP request.

Now that I think about it, I remember I wanted to change it previously, but it was tricky…

**Comment by neilhem** [comment]
Maybe we should remove integration with boostrap plugins on generator bootstrap & include them via wiredep

**Comment by silvenon** [comment]
@neilhem good idea!

**Comment by illycz** [comment]
What is the "right" solution for now, if I don't want another request?

Thanks

**Comment by silvenon** [comment]
One way is to include them all in the `vendor.js` block:

```html
<!-- build:js scripts/vendor.js -->
<!-- bower:js -->
<!-- endbower -->
<!-- insert Bootstrap plugins here -->
<!-- endbuild -->
```

**Comment by illycz** [comment]
wiredep remove this hardcoded files in next running... :(

**Comment by neilhem** [comment]
Wiredep removes everything between
```html
<!-- bower:js -->wiredep injected scripts here<!-- endbower -->
```
Scripts which you don't want to be affected by wiredep you should put like this:
```html
<!-- build:js scripts/vendor.js -->
<!-- bower:js -->
<!-- endbower -->
wiredep will not update these scripts
<!-- endbuild -->
```

**Comment by UlisesGascon** [comment]
I assume that this bug was fixed in #737. If not we can re-open it again 😃
